src/styles/global.css

```
@@ -4,6 +4,8 @@
 @plugin "flowbite/plugin";
 @source "../../node_modules/flowbite";
 
+@custom-variant dark (&:where(.dark, .dark *));
```

Stylelint (17.7.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)